common-util/AbiAndAddresses/tokenomics.jsx

```diff
@@ -1,6 +1,6 @@
-export const TOKENOMICS_ADDRESS_MAINNET = '0xc096362fa6f4A4B1a9ea68b1043416f3381ce300';
+export const TOKENOMICS_ADDRESS_MAINNET = '0xD599ED4a5c45Fa66AA3C474b00701Fd452a2C30e';
```

Comment:
You can find the abi on this page: https://etherscan.io/address/0xD599ED4a5c45Fa66AA3C474b00701Fd452a2C30e#code

Reply:
I believe the ABI is extracted correctly via the .json files in the tokenomics repo, however the address must stay the same as the proxy contract has not changed.